### DISCUSSION TITLE
FLASH-CAT

### DISCUSSION THREAD

**Discussion by h7899** [firemodels/fds#13006] [Q&A]
I AM A BEGINNER, PLS HOW CAN I GET A FLASH-CAT MODEL CASE.

**Comment by Er9y714** [comment]
[See the discussion](https://github.com/firemodels/fds/discussions/12861#discussioncomment-9591964)

**Reply by drjfloyd** [reply]
It is an empircal model for developing a time dependent heat release rate for cable tray fires for use in fire probabilisitic risk assessment for nuclear power plants. It is documented in NUREG/CR-7010.
